src/components/ErrorBox.tsx

```diff
@@ -0,0 +1,32 @@
+import React from "react"
+import { APIError } from "../data/types"
+
+interface ErrorBoxParams {
+  title: string
+  refetch: () => void
+  error: APIError
+}
+
+function ErrorBox({ title, error, refetch }: ErrorBoxParams) {
+  const message = error instanceof Error ? error.message : "Unknown error"
+
+  return (
+    <div className="error-box-container">
+      <div className="error-container">
```

Comment:
The double div seems unnecessary, you could probably make do with just a single `error-box` div.

Reply:
You are right, the way I was going to style the component needed it but it does not anymore